ext/rugged/rugged_tree.c

```diff
@@ -489,6 +489,8 @@ void rugged_parse_merge_options(git_merge_options *opts, VALUE rb_options)
 
 		if (RTEST(rb_hash_aref(rb_options, CSTR2SYM("renames")))) {
 			opts->flags |= GIT_MERGE_FIND_RENAMES;
+		} else {
+			opts->flags &= ~GIT_MERGE_FIND_RENAMES;
```

Comment:
`!RTEST` won't work here because `nil` also evaluates to `0` with `RTEST` and not specifying this option would clear it 😖.